NightscoutUploadKit/NightscoutUploader.swift

```
@@ -142,12 +142,17 @@ public class NightscoutUploader: NSObject {
             nsStatus["uploader"] = ["battery":uploaderDevice.batteryLevel * 100]
         }
         
-        let pumpDate = TimeFormat.timestampStr(status.pumpDateComponents)
+        guard let pumpDate = status.pumpDateComponents.date else {
+            NSLog("Pump date not set (or timezone missing) in pump status message!")
```

Maybe we can circle back and `throw` instead of logging here?